litreview.md

```diff
@@ -0,0 +1,50 @@
+# Overview
+
+Continuous ATO (C-ATO) is an organizational initiative to automate security compliance activity. In collaboration with industry, NIST is developing the Open Security Controls Assessment Language (OSCAL), a set of formats expressed in XML, JSON, and YAML. These formats provide machine-readable representations of control catalogs, control baselines, system security plans, and assessment plans and results. A security team will likely only work with OSCAL indirectly but through a tool. The purpose of OSCAL is a standardized, machine-readable format to share artifacts between Governance, Risk, and Compliance (GRC) and software automation tools or programs.
```

Comment:
Are we going to reference [the final DoD memo]() or the related NIST whitepapers where these concepts were originally defined? Kessel Run staff helped establish the DoD interpretation, but it is rooted in whitepapers presenting a modern interpretation of C-ATO as a method of applying RMF or rather it was really Continuous RMF. These concepts seem important to define and source in this document as key assumptions, no?